serialization/src/main/kotlin/net/corda/serialization/internal/amqp/PropertyDescriptor.kt

```
@@ -88,17 +87,15 @@ private val propertyMethodRegex = Regex("(?<type>get|set|is)(?<var>\\p{Lu}.*)")
  * take a single parameter of a type compatible with exampleProperty and isExampleProperty must
  * return a boolean
  */
-internal fun Class<out Any?>.propertyDescriptors(warnInvalid: Boolean = true): Map<String, PropertyDescriptor> {
+internal fun Class<out Any?>.propertyDescriptors(): Map<String, PropertyDescriptor> {
     val fieldProperties = superclassChain().declaredFields().byFieldName()
 
     return superclassChain().declaredMethods()
             .thatArePublic()
             .thatArePropertyMethods()
             .withValidSignature()
             .byNameAndClassifier(fieldProperties.keys)
-            .toClassProperties(fieldProperties).run {
```

`warnInvalid` needs to stay, because its absence suppresses an exception that we definitely don't want thrown when enumerating the properties of non-composable types, but do want thrown if a composable type has an invalid getter/setter pair (for example).